frontend/src/index.html

```diff
@@ -86,10 +86,10 @@ <h4 class="text-sm font-semibold uppercase md:text-base">Audio Recordings</h4>
     <div class="mt-4 grid grid-cols-1 gap-2 md:grid-cols-2">
       {% for audio in audios offset: 25 limit: 8 %}
         <a href="./audios/{{ audio.objectID }}/">
-          <div class="grid  grid-cols-6 rounded-md border border-indigo-300 hover:!bg-yellow-100 hover:!bg-opacity-50 md:grid-cols-8">
-            <div class="col-span-1 ml-2">
+          <div class="flex items-center rounded-md border border-indigo-300 hover:!bg-yellow-100 hover:!bg-opacity-50">
```

Comment:
- We don't use important specifiers.
- Remove hover, it's too flashy.
- Why `indigo`? We don't use this color elsewhere.
- This `div` is also redundant, apply styles directly to the `a` element.

Reply:
updated as per review....